src/Components/ArticlePage.jsx

```diff
@@ -15,12 +16,12 @@ export default function ArticlePage({}) {
   }, []);
 
   return isLoading ? (
-    <h6>Article is loading</h6>
+    <p></p>
```

Comment:
wait bring loading back !!

Reply:
Hi Pippa, there is an isLoading on the app.jsx as well and when i set the is loading on the article page to give any massage it causes a double isloading message. I need to fix that